doc/lua_api.md

```diff
@@ -8824,6 +8906,18 @@ Used by `minetest.register_node`.
     -- Special textures of node; used rarely.
     -- List can be shortened to needed length.
 
+    -- See "Variants"
+	  variants = {
+            [variant number] = {
+                tiles = {tile definition 1, def2, def3, def4, def5, def6},
+
+                overlay_tiles = {def1, def2, def3, def4, def5, def6},
+
+                special_tiles = {def1, def2},
+            },
+            ...
+        },
+
```

Comment:
Maybe show the example of using the index `[0]`, followed by array items.
Because I think it should be reminded here that variants have 0-based indices.